Sources/NIO/SelectableEventLoop.swift

```diff
@@ -56,7 +56,7 @@ internal final class SelectableEventLoop: EventLoop {
     /* private but tests */ internal let _selector: NIO.Selector<NIORegistration>
     private let thread: NIOThread
     @usableFromInline
-    internal var _scheduledTasks = PriorityQueue<ScheduledTask>(ascending: true)
+    internal var _scheduledTasks: PriorityQueue<ScheduledTask>? = PriorityQueue<ScheduledTask>(ascending: true)
```

Comment:
@realdoug one question here: Why do we make this optional instead of say adding a state to the `InternalState`?

Reply:
That was my suggestion: https://github.com/apple/swift-nio/pull/1395#discussion_r380049404